academy/0.0-B9lab-Blockchains/3_managed.md

```diff
@@ -0,0 +1,142 @@
+
+<HighlightBox type="info">
+
+**Ether** is the **native cryptocurrency** of the Ethereum platform, as Bitcoin is the native cryptocurrency of the Bitcoin blockchain.
+
+</HighlightBox>
+
+The protocol defines the **Gas cost** for each operation in the EVM. The halting problem is resolved by attaching financial costs to every step in a program and by limiting the funds available. Transactions that do not compute successfully and are not on budget are rejected.
+
+In Ethereum, **smart contracts** are stored on every single node - they are public. In practice, a smart contract in the EVM is an autonomous agent with an internal account. The most popular language for writing such a contract is **Solidity**.
+
+<ExpansionPanel title="Faster block time">
+
+Ethereum has a much faster block time than Bitcoin - currently it is around 15 seconds. 
+
+Ethereum implements a variation of Bitcoin's PoW consensus algorithm called Ethash, which is intended to be [ASIC-resistant and GPU friendly](https://github.com/ethereum/wiki/wiki/Ethash-Design-Rationale). ASIC-resistance is meant to discourage unwanted concentrations of mining power; in other words, encourage decentralization. Core developers of the Ethereum project plan to switch to the **proof-of-stake (PoS)** consensus algorithm in the future.
+    
+These useful innovations are not the key innovation that allows Ethereum to operate with faster block times. 
+
+A well-understood challenge related to reduced block time stems from network latency. Since nodes do not learn about newly discovered blocks simultaneously, at any given time a portion of the miners will be working on already solved (i.e. old) blocks. If these miners find a solution, they might not be rewarded and a "fork" of the chain results, in which rival chains with a common true history spread through the network. The rate at which such transient forks occur increases predictably with shorter block times. 
+
+Another way of understanding this is to consider that at any point in time, a predictable percentage of the total mining hardware in the network will be working on the wrong chain, because of latency, and this percentage increases with shorter block times. It is not beneficial to the network or the miners to waste hardware resources that are intended to secure the network. 
+
+While Bitcoin addresses this efficiency issue with a relatively long block time of 10 minutes, Ethereum addresses this concern with a partial reward strategy: valid blocks that aren't ultimately included in the canonical chain (because another chain becomes longer) are still included on the side and known as **uncles**. Miners of uncles receive a smaller reward than regular block miners. 
```

Comment:
```suggestion
While Bitcoin addresses this efficiency issue with a relatively long block time of 10 minutes, Ethereum addresses this concern with a partial reward strategy: valid blocks that aren't ultimately included in the canonical chain (because another chain becomes longer) can still be included on the side and known as **uncles**. Miners of uncles receive a smaller reward than regular block miners. 
```